cmd/manager/main.go

```diff
@@ -271,6 +272,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = (&atlasbackupcompliancepolicy.AtlasBackupCompliancePolicyReconciler{
```

Comment:
You also need to `Watch` for backup compliance policies in the AtlasProjectReconciler. Currently, changes done in a backup compliance policy won't trigger a change on the atlas sinde.